curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153908a366afb4d57185c8d.md

```diff
@@ -7,16 +7,25 @@ dashedName: step-12
 
 # --description--
 
-Flexbox can be thought of as having two axes, the main axis and the cross axis. The main axis is determined by the `flex-direction` property. If `flex-direction` is set to `row` or `row-reverse`, the main axis is horizontal. If `flex-direction` is set to `column` or `column-reverse`, the main axis is vertical.
+Flexbox has a main and cross axis. The main axis is defined by the `flex-direction` property, which has four possible values:
 
-Give your `#gallery` selector a `flex-direction` property set to `row`.
+- `row` (default): horizontal axis with flex items from left to right
+- `row-reverse`: horizontal axis with flex items from right to left
+- `column`: vertical axis with flex items from top to bottom
+- `column-reverse`: vertical axis with flex items from bottom to top
+
+**Note**: The axes and directions will be different depending on the text direction. The values shown are for a left-to-right text direction.
+
+Try the different values to see how they affect the layout.
+
+When you are done, set an explicit `flex-direction` of `row` on the `.gallery` element.
```

Comment:
there is no point of setting `flex-direction: row`, it will be redundant code, and I think it's frowned upon to have redundant code.
 
```suggestion
set an explicit `flex-direction` of `row-reverse` on the `.gallery` element.
```

Reply:
It isn't so much redundant as it is explicit.

I agree that explicitly setting it to what is the default value is a little odd but I also think reversing the order for no reason and without giving a reason is as well.

I can see both sides of the argument but I really think we shouldn't use reverse (or `order`) unless there is a very good reason for it. It is frowned upon from an accessibility perspective.

Reply:
> I agree that explicitly setting it to what is the default value is a little odd

Honestly, I think it's pretty normal. Explicitly setting a value ensures that it's not accidentally changed by browser defaults, or CSS libraries, and such.

Reply:
> explicitly setting it to what is the default value is a little odd (but actually not that uncommon)

>  by browser defaults, or CSS libraries, and such.

I am convinced, should we mention why in the steps?

Reply:
@Sboonny I'd prefer not to do that. At least, not in this PR. I think we are already doing too much in this single PR.

Reply:
@naomi-lgbt True, I did edit the comment to say that it isn't that uncommon. But without explaining why you would want to set the default explicitly I can see us getting asked about it.

Reply:
I will open issue about it after this pull request merge.

If I don't lose the note 😅